src/main/java/wolox/training/models/Book.java

```diff
@@ -134,6 +142,7 @@ public Integer getPages() {
     }
 
     public void setPages(Integer pages) {
+        Preconditions.checkNotNull(pages);
```

Comment:
you can check that the pages be Integer and not string, or that the pages ≠ 0, for example

Reply:
I added validation in this field.